addons/mail/static/src/models/message_reaction_group_partner.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'MessageReactionGroupPartner',
+    fields: {
+        messageContextViewOwner: one('MessageContextMenu', {
+            identifying: true,
+            inverse: 'messageContextReactionPartners',
+        }),
+        messageReactionGroup: one('MessageReactionGroup', {
+            identifying: true,
+            inverse: 'messageContextReactionPartners'
+        }),
```

Comment:
Could just rename model to `(Inline)PersonaView`.

Right now it's quite heavily reliant on `messageReactionGroup`.
We'll likely adjust this model in the future and/or following reviews so that it's just a record that models an inline visual of persona.

Would define some essential (computed) fields like `persona`, and shape it in a way to have many contexts of use.
For example, `ComposerSuggestionView` does have a part for `Partner`, whose visual is basically the same as this `(Inline)PersonaView`

Reply:
Should I do the last part or just change the naming for now?

Reply:
I would define `InlinePersonaView` for this model, and have its identifying field be `persona` and `owner`. The `owner` targeted model has `identifyingMode: 'xor'`, to list the different contexts of using an `InlinePersonaView`.
Right now just used in `MessageContextMenu`, but I see it being used in some other places very soon, like `ComposerSuggestionView` when it's an `@`-mention.